specs/checking_account_spec.rb

```diff
@@ -1,80 +1,165 @@
 require 'minitest/autorun'
 require 'minitest/reporters'
 require 'minitest/skip_dsl'
+require_relative '../lib/checking_account'
+require_relative '../lib/account'
 
-# TODO: uncomment the next line once you start wave 3 and add lib/checking_account.rb
-# require_relative '../lib/checking_account'
 
-# Because a CheckingAccount is a kind
-# of Account, and we've already tested a bunch of functionality
-# on Account, we effectively get all that testing for free!
-# Here we'll only test things that are different.
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "CheckingAccount" do
+describe "CheckingAccount" do
+
   describe "#initialize" do
-    # Check that a CheckingAccount is in fact a kind of account
+
     it "Is a kind of Account" do
-      account = Bank::CheckingAccount.new(12345, 100.0)
+      account = Bank::CheckingAccount.new(12345, 10000)
       account.must_be_kind_of Bank::Account
     end
   end
 
   describe "#withdraw" do
```

Comment:
Good work on these tests. There's a bunch of complex logic to verify here, and I feel like you've done a solid job of covering it.